src/js/containers/SubMenu.js

```diff
@@ -38,33 +46,47 @@ const mapStateToProps = (state) => {
     // The app isn't performing an interaction, so pass the sub menu items 
     var menu = app.menu
     var activeSubMenu = app.activeSubMenu
-    if (!activeSubMenu || !menu) {
-        return {data: [], isPerformingInteraction: false, theme: theme}
-    }
-    data = menu.find((test) => {
-        return test.menuID === activeSubMenu
-    }).subMenu.map((command) => {
+    var ddState = state.ddState;
+    var subMenuData = SubmenuDeepFind(menu, activeSubMenu, 0).subMenu.subMenu.map((command, index) => {
```

Comment:
The HMI crashes on this line when I select anything in a TILED submenu or any command within a submenu

```
TypeError: Cannot read property 'subMenu' of null
    at Function.Mn [as mapToProps] (SubMenu.js:50)
    at r (wrapMapToProps.js:41)
    at Function.r.mapToProps (wrapMapToProps.js:50)
    at r (wrapMapToProps.js:41)
    at selectorFactory.js:22
    at connectAdvanced.js:319
    at Object.useMemo (react-dom.production.min.js:4005)
    at t.useMemo (react.production.min.js:411)
    at O (connectAdvanced.js:304)
    at qi (react-dom.production.min.js:3667)
```

Reply:
Fixed in 
dce1586